x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/cli/SelectTestCase.java

```diff
@@ -37,4 +37,14 @@ public void testSelectWithWhere() throws IOException {
         assertThat(readLine(), RegexMatcher.matches("\\s*2\\s*\\|\\s*test_value2\\s*"));
         assertEquals("", readLine());
     }
+
```

Comment:
I would create a test class just for this command. And, apart from the test you have below, I'd test the `false` scenario as well (the one that generates an error).

Reply:
Done